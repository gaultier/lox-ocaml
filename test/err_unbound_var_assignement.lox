a=1; // expect: Assigning unbound variable a
