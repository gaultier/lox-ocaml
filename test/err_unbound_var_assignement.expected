Assigning unbound variable `a` to `1.`
